docs/contributor/01-00-installation.md

```diff
@@ -89,153 +88,72 @@ kubectl apply -f config/samples/operator_v1alpha1_istio.yaml
-- Kyma CR deployed and Ready! (3.121s)
-
-Kyma is installed in version:	alpha deployment with lifecycle-manager
-Kyma installation took:		1m20s
-
-Happy Kyma-ing! :)
+export MODULE_REGISTRY=registry.localhost:5001/unsigned
 ```
 
-Kyma installation is ready, but no module is activated yet.
+5. Navigate to the directory with the cloned project and build the Istio module.
 
 ```bash
-kubectl get kymas.operator.kyma-project.io -A
-NAMESPACE     NAME           STATE   AGE
-kyma-system   default-kyma   Ready   37s
+make module-build
 ```
 
-2. Apply `template.yaml` to register Istio as a module known for Lifecycle Manager.
-
-Istio Module is a known module, but it is not activated.
+6. Apply the ModuleTemplate generated by the build:  
 
 ```bash
-kubectl get moduletemplates.operator.kyma-project.io -A 
-NAMESPACE    NAME                   AGE
-kcp-system   moduletemplate-istio   2m24s
+kubectl apply -f template.yaml
 ```
 
-3. Give Lifecycle Manager permission to install CustomResourceDefinition (CRD) cluster-wide.
-
->**NOTE:** This is a temporary workaround only required in the single-cluster mode.
-
-Lifecycle Manager must be able to apply CRDs to install modules. When it operates in remote mode, where the control plane manages remote clusters, it receives an administrative kubeconfig that allows it to target the remote cluster and apply CRDs. However, when it operates in local mode (single-cluster mode), it uses Service Account and does not have permission to create CRDs by default.
-
-Run the following command to make sure the Lifecycle Manager's Service Account is granted an administrative role:
+7. Enable the Istio module. 
 
 ```bash
-kubectl edit clusterrole lifecycle-manager-manager-role
+kyma alpha enable module istio -c alpha
 ```
 
-Add the following configuration:
+### Install with artifacts built for the `main` branch of the Istio repository
 
-```yaml
-- apiGroups:                                                                                                                  
-  - "*"                                                                                                                       
-  resources:                                                                                                               
-  - "*"                                                                                                                       
-  verbs:                                                                                                                      
-  - "*"
-```
-
-4. Enable the Istio module.
+1. Provision the k3d cluster.
 
 ```bash
-kyma alpha enable module istio -c alpha
+kyma provision k3d
 ```
 
-## Install with artifacts built for the `main` branch of the Istio repository
-
-You can install the Istio module using the artifacts that are created by `post-istio-module-build` job. To do so, follow these steps:
+2. Install Kyma with Lifecycle Manager in the target cluster.
```

Comment:
This command will only Install Lifecycle Manager, no other components are installed. That's why I think it can be confusing to write "Install Kyma".
```suggestion
2. Install Lifecycle Manager in the target cluster.
```